app/services/CheckoutService.scala

```diff
@@ -80,7 +80,7 @@ class CheckoutService(identityService: IdentityService,
       defaultPaymentDelay = CheckoutService.paymentDelay(subscriptionData.productData, zuoraProperties)
       paymentMethod <- EitherT(attachPaymentMethodToStripeCustomer(payment, purchaserIds))
       subscribe = createSubscribeRequest(personalData, soldToContact, requestData, plan, purchaserIds, paymentMethod, payment.makeAccount, Some(defaultPaymentDelay))
-      validPromotion = promoCode.flatMap(promoService.validate[NewUsers](_, personalData.address.country.getOrElse(Country.UK), subscriptionData.productRatePlanId).toOption)
+      validPromotion = promoCode.flatMap(promoService.validate[PromoContext.Renewal](_, personalData.address.country.getOrElse(Country.UK), subscriptionData.productRatePlanId).toOption)
```

Comment:
For Guardian Weekly subs (acquisition AND renewal), the country being validated for the promotion should be the delivery country not the billing country.

Happy if this is created as a new ticket.